### ISSUE TITLE
How GEM2 use 3d information 

### ISSUE THREAD

**Issue by tiendat104** [PaddlePaddle/PaddleHelix#225]
Hi
I recently read your paper "GEM-2: Next Generation Molecular Property Prediction Network by Modeling Full-range Many-body Interactions" and I'm quite impressed by the performance of GEM2 on PCQM4Mv2.
However, I have some difficulties in understanding its implementation. 
In the method __call__ of class OptimusTransformerFn from https://github.com/PaddlePaddle/PaddleHelix/blob/dev/apps/pretrained_compound/ChemRL/GEM-2/src/featurizer.py
I see two methods to compute 3d coordiantes for each molecule: (1) raw3d and (2) rdkit3d 
The first method seems to load the 3d information provided by PCQM4Mv2 dataset, so it only applies for training set.
The second method seems to use some built-in algorithm of rdkit to compute 3d information and can apply for both training and valid, test set. 
So here are my questions: 
(1) For the result reported in your paper, which method you use to compute 3d information? raw3d or rdkit3d?
(2) Does GEM2 use 3d information during inference on valid and test set? Or it just turn off 3d information?
(3) If possible, can you provide the pretrained weight for the GEM2 model reported in the paper?
Thank you!

**Comment by Noisyntrain** [comment]
1. In the result reported in our paper we are using the rdkit3d information.
2. Yes, GEM2 uses 3d information during inference.
3. Sure, we will update the  pretrained weight soon.
Thank you for your interest in GEM2 :）

**Comment by tiendat104** [comment]
Great answer. Thank you!

**Comment by microbenh** [comment]
Is that the only 3d infomation used in GEM2 is **the pair distance** of  table 3 in the paper?
![image](https://user-images.githubusercontent.com/37615923/203892318-e827f621-0579-451e-bf27-ea4bd82820d8.png)
